Dockerfile.builder

```diff
@@ -18,13 +19,13 @@ RUN yum --enablerepo=centosplus install -y --quiet \
       postgresql \
     && yum clean all
 
-RUN test -n $USE_GO_VERSION_FROM_WEBSITE \
-    && cd /tmp \
+RUN if [[ "$USE_GO_VERSION_FROM_WEBSITE" = 1 ]]; then cd /tmp \
```

Comment:
Wish we could have had unit tests for these :)

Reply:
Me too :)